tests/test_enterprise/test_signals.py

```diff
@@ -802,6 +802,7 @@ def test_create_catalog(self, api_client_mock):
             enterprise_catalog.enabled_course_modes,
             enterprise_catalog.publish_audit_enrollment_urls
         )
+        api_client_mock.return_value.refresh_catalogs.assert_called_with([enterprise_catalog])
```

Comment:
Nit: could these be `assert_called_once_with()`?

Reply:
Indeed they can. I'll throw in a few resets so the calls don't stack from test to test

Reply:
Hrmph ok, the reset doesn't just clear the call count it fully resets the mock. I think I'm going to pass on this for now to avoid mocking it for the whole class and resetting the mock after each one